base/markdown/Julia/interp.jl

```diff
@@ -39,7 +39,7 @@ end
 
 toexpr(x) = x
 
-toexpr(xs::Vector{Any}) = Expr(:cell1d, map(toexpr, xs)...)
+toexpr(xs::Vector{Any}) = Expr(:vectany, map(toexpr, xs)...)
```

Comment:
Ok, I guess since this was deprecated in 0.4 we should actually remove it completely, and use `Expr(:ref, :Any, ...)` instead.